dts/bindings/cpu/cpu.yaml

```diff
@@ -6,14 +6,18 @@
 include: base.yaml
 
 properties:
+    timebase-frequency:
+      type: int
+      required: false
+      description: Frequency of the timebase in Hz.
```

Comment:
`timebase-frequency` is semantically equivalent to `CONFIG_SYS_CLOCK_TICKS_PER_SEC` in Zephyr. Unless we plan to switch to using it instead of `CONFIG_SYS_CLOCK_TICKS_PER_SEC`, we should probably hold off on adding it.

@galak any comments?

Reply:
> timebase-frequency is semantically equivalent to CONFIG_SYS_CLOCK_TICKS_PER_SEC in Zephyr.

I think, just this justify add this property. In general, I see changes like this be incremental. I'm only proposing add the non required property right now, nothing more.

I was wondering if this could not be a simple default at Kconfig, that gets value from devicetree.